src/test/convertion.test.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('assert');
+const sheetUtils = require('../main/lib/ccd-spreadsheet-utils');
+
+describe('JsonHelper', () => {
+  const jsonWithNumbers = ()=>[
```

Comment:
Since these are methods that create instances let's add some verb in there. Something like `createJsonWithNumbers` would work better for me.

Same applies to second factory method.